source/billing.txt

```diff
@@ -15,24 +15,24 @@ Billing
 Overview
 --------
 
-{+service+} applications are billed according to how much they're used on a
+Atlas App Services applications are billed according to how much they're used on a
 monthly basis. There are four independently-measured usage types that determine
 how much you pay each month:
 
-- :ref:`{+service-short+} Requests <billing-requests>`, which measures the number of
-  requests that {+service+} sends and receives from client
+- :ref:`App Services Requests <billing-requests>`, which measures the number of
+  requests that App Services sends and receives from client
   applications and external services. Requests include function calls,
   trigger executions, and sync updates, but exclude user authentication
   and blocked or invalid requests.
 
-- :ref:`{+service-short+} Compute <billing-compute>`, which measures the runtime and memory
+- :ref:`App Services Compute <billing-compute>`, which measures the runtime and memory
   usage of all requests excluding sync and authentication.
 
-- :ref:`{+sync+} <billing-sync>`, which measures the total amount of time in
+- :ref:`Atlas Device Sync <billing-sync>`, which measures the total amount of time in
   which a client application is actively syncing.
 
 - :ref:`Data Transfer <billing-data-transfer>`, which measures the amount of
-  data that {+service+} sends to external services and client applications.
+  data that Atlas App Services sends to external services and client applications.
```

Comment:
As this is not the first time the product name is referenced on the page, shouldn't this just be "App Services" instead of "Atlas App Services"?